pypeclub/OpenPype
# Nuke: Adding `still` image family workflow

Adding Still image nuke write rendering workflow from creator via publisher to loader.

# Testing notes
1. start Openpype with settings and make sure you are not having any project overides on Anatomy imageio and nuke loadImage
2. select any node in your testing workfile and go to OpenPype Menu / Create / `Create Write Still Image`. Notice the name of subset will be created regarding actual frame on timeline
3. Publish first with local setting in render knob and then with already rendered frame. 
4. Test Validator for rendered frames if you set already rendered frame and it was not rendered yet, it should be alarming your
5. Check published subset in loader and load it into scene. 
6. in Manager set to lower version then it is actually loaded into your workfile. 
7. all should be working as expected ;)
